src/app/pages/home/home.tsx

```diff
@@ -47,7 +52,13 @@ function HomeContainer({ account }: HomeContainerProps) {
       actions={<HomeActions />}
     >
       <HomeTabs>
-        <Outlet context={{ address: account.address }} />
+        <>
```

Comment:
I need to declare `Routes` here so I can manipulate the `location` depending on whether `backgroundLocation` is populated or not. 

`backgroundLocation` is the route above which the modal should appear.

Reply:
Isn't it possible to also read location in app-routes to see if bg location is defined? Nw if not, just curious

Reply:
We can read the location there but the issue I struggled with is how to then feed said location into how we are creating the routes. 

We are using `createHashRouter` and `createRoutesFromElements` and I was unable to figure out how to pass the `location` there. 

Eventually I came up with the idea of using `<Routes` nested in `<Home` so I could then trick the `location`.

We can probably improve on that structure with more time and I am open to suggestions but I didn't want to start refactoring the whole of `app-routes`